src/main/java/signatures/Version1.java

```diff
@@ -0,0 +1,67 @@
+package signatures;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.format.DateTimeFormatter;
+import org.joda.time.format.ISODateTimeFormat;
+
+public class Version1 {
+
+    private final String role;
+    private final String signingTime;
+    private final String cfInstanceCertContents;
+    private final String signature;
+
+    public Version1(String role) {
+        // Get the components of the string we'll sign.
+        String cfInstanceCert = Utilities.getFileBodyAt("CF_INSTANCE_CERT");
+
+        // Get the current time, UTC, formatted like so: 2019-05-20T22:08:40Z.
+        DateTime signingTime = DateTime.now().withZone(DateTimeZone.UTC);
+        DateTimeFormatter fmt = ISODateTimeFormat.dateTimeNoMillis();
+        String signingTimeStr = fmt.print(signingTime);
+
+        // Concatenate them to build our string to sign.
+        String stringToSign = Utilities.getStringToSign(role, signingTimeStr, cfInstanceCert);
+        System.out.println("*************************SIGNING*************************");
```

Comment:
We will want to get rid of these printlns :-)

Reply:
OK! :-) I'd left them out in case people were debugging but you're right, that would get pretty weird if the tool were ever imported and they had a logger they wanted to use. I don't really want to deal with logging on this PR (or hopefully ever). :-)